recipes/diffeqpy/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set name = "diffeqpy" %}
+{% set version = "2.4.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/diffeqpy-{{ version }}.tar.gz
+  sha256: 4cd093feaee6b70e4f4207ed284f35ff0338e614f5f4d2fd327b854dc8c7f2c0
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+
+requirements:
+  host:
+    - python >=3.6
+    - pip
+  run:
+    - python >=3.6
+    - pyjuliacall >=0.9.14
+    - jill
+    # jill 0.11.5 has requirement fire<0.6.0,>=0.5.0, but you have fire 0.6.0.
```

Comment:
I opened an issue up-stream to fix this https://github.com/conda-forge/jill-feedstock/pull/9